react/src/lib/EventOverlay/index.js

```diff
@@ -367,16 +367,19 @@ class EventOverlay extends React.Component {
   handleKeyDown = e => {
     if (!this.props.isOpen) return;
     if (e.keyCode === 27) return this.handleClickAway(e);
-    const eventTarget = eventPath(e)[0];
-    const anchorNode = ReactDOM.findDOMNode(this.props.anchorNode);
-
-    return (
-      this.container &&
-      anchorNode &&
-      !anchorNode.contains(eventTarget) &&
-      !ReactDOM.findDOMNode(this.container).contains(eventTarget) &&
-      this.handleClickAway(e)
-    );
+    
+    if (!this.props.disableCloseOnEnterKey) {
```

Comment:
Where do we specifically check for the enter key?

Reply:
I see what you mean, I should probably separate it out
Need to check for ENTER/SPACE. Will change soon :)